recyclerviewfastscroller/src/main/java/com/qtalk/recyclerviewfastscroller/RecyclerViewFastScroller.kt

```diff
@@ -324,6 +329,9 @@ class RecyclerViewFastScroller @JvmOverloads constructor(
                 (loadDrawableFromAttributes(R.styleable.RecyclerViewFastScroller_handleDrawable)
                     ?: ContextCompat.getDrawable(context, Defaults.handleDrawableInt))
 
+            hideHandleAfter =
```

Comment:
let change this to: 
```suggestion
            handleVisibilityDuration =
```
let's also change the style attribute